src/java.base/share/classes/java/io/ByteArrayInputStream.java

```diff
@@ -158,6 +158,11 @@ public synchronized int read() {
      * <p>
      * This {@code read} method cannot block.
      *
+     * @apiNote
+     * Unlike the {@link InputStream#read(byte[],int,int) equivalent method}
+     * of {@code InputStream}, this method returns {@code -1} instead of zero
```

Comment:
I think I would say superclass method instead of equivalent method.
Or prehaps overridden method.

Reply:
I think overridden would work best.

Reply:
I think it needs to be normative text rather than an apiNote.